packages/waku/src/lib/middleware/rsc/ssr.ts

```diff
@@ -72,13 +72,13 @@ export const loadServerFile = async (
 
 // FIXME this is very hacky
 const createTranspiler = async (cleanupFns: Set<() => void>) => {
-  return (file: string, name: string) => {
+  return (filePath: string, name: string) => {
     const temp = path.resolve(
       `.temp-${crypto.randomBytes(8).toString('hex')}.js`,
     );
     const code = `
-import { loadServerFile } from '${url.fileURLToPath(import.meta.url)}';
-const { ${name} } = await loadServerFile('${file}', 'dev');
+const { loadServerFile } = await import('${import.meta.url}');
```

Comment:
Do you know why import statement doesn't work?

Reply:
I think there might because ``

Reply:
Yep. fileURLToPath will be `C:\Users\himse\Code\waku\index.mjs` `\` is invalid in strict mode

Reply:
It no longer uses fileURLToPath, so does it work with import statement?
Or, does it still complain in strict mode?

Reply:
No I still have such error

```
You did not run Node.js with the `--conditions react-server` flag. Any "react-server" override will only work with ESM imports.
SyntaxError: Octal escape sequences are not allowed in strict mode.
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:118:18)
    at callTranslator (node:internal/modules/esm/loader:273:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:278:30)
    at async link (node:internal/modules/esm/module_job:76:21)
```

Reply:
Do you mean there's a regression?

Reply:
I think so

Reply:
I give a patch that can make code works on my windows

Reply:
nice! 29f74aa is what I did. Does it work? which do you prefer?